src/serve/index.js

```diff
@@ -54,6 +50,8 @@ exports.serveWeb = function (opts, cb) {
   app.use('/compile/', importMiddleware(getPath('static/')));
 
   // serve static files
+  var devkitPath = path.resolve(__dirname, '..', '..');
+  app.use('/devkit/', express.static(devkitPath));
```

Comment:
Serving the entire thing because its part of the available path ingame, I
think. If we don't need devkit on the games path, we don't have to serve
the whole thing (or if there are specific subpaths)
On Jul 29, 2015 1:23 PM, "Martin Hunt" <notifications@github.com> wrote:

> In src/serve/index.js
> <https://github.com/gameclosure/devkit/pull/238#discussion_r35807066>:
>
> > @@ -54,6 +50,8 @@ exports.serveWeb = function (opts, cb) {
> >    app.use('/compile/', importMiddleware(getPath('static/')));
> >
> >    // serve static files
> > +  var devkitPath = path.resolve(__dirname, '..', '..');
> > +  app.use('/devkit/', express.static(devkitPath));
>
> this feels a little excessive, do we need to serve the entire devkit
> directory?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gameclosure/devkit/pull/238/files#r35807066>.
>